File: split/printglue.pas
procedure printglue(d: scaled; order: integer; s: strnumber);
begin
  printscaled(d);
  if (order < 0) or (order > 3) then
    print(310)
  else if order > 0 then
  begin
    print(
      311);
    while order > 1 do
    begin
      printchar(108);
      order := order - 1;
    end;
  end
  else if s <> 0 then
    print(s);
end;

File: cpp/printglue.cpp
#include "printglue.h"
#include "print.h"
#include "printchar.h"
#include "printscaled.h"
#include "texte.h"

void printglue(scaled d, int order, const std::string &s)
{
	printscaled(d);
	if (order < 0 || order > 3)
		print("foul");
	else 
		if (order > 0)
		{
			print("fil");
			for (;order > 1; order--)
				printchar('l');
		}
		else 
			if (txt(s))
				print(s);
}
